homeassistant/components/aranet/strings.json

```diff
@@ -21,5 +21,12 @@
       "no_devices_found": "No unconfigured Aranet devices found.",
       "outdated_version": "This device is using outdated firmware. Please update it to at least v1.2.0 and try again."
     }
+  },
+  "entity": {
+    "sensor": {
+      "update_interval": {
+        "name": "Update interval"
```

Comment:
Most sensor names I've seen have used title case rather than sentence case: should "interval" be capitalized? (It is in the code you're replacing, as well.)